cookbooks/aws-parallelcluster-slurm/resources/munge/partial/_munge_actions.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster-slurm
+# Recipe:: install_munge
+#
+# Copyright:: 2013-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Munge
+munge_version = '0.5.14'
+munge_url = "https://github.com/dun/munge/archive/munge-#{munge_version}.tar.gz"
+munge_tarball = "#{node['cluster']['sources_dir']}/munge-#{munge_version}.tar.gz"
+munge_user = 'munge'
+munge_user_id = node['cluster']['reserved_base_uid'] + 2
+munge_group = munge_user
+munge_group_id = munge_user_id
+
+action :setup do
```

Comment:
I agree that `setup` is a very broad term that can be replaced with a more descriptive name, however as you stated in the following comment is not yet clear `what the default action should be` so I'll keep this name since it is the `standard` we agreed upon for the refactoring and leave the "optimizations" to a later time when the usage requirement will be more clear.

Reply:
Resolved for now.